addons/pinniped/config-controller/controllers/v3_controller.go

```diff
@@ -75,8 +76,9 @@ func (c *PinnipedV3Controller) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 
 		for i := range secrets.Items {
+			log.Info("Reconciling secret", "secret", secrets.Items[i].Name, "namespace", secrets.Items[i].Namespace)
 			if err := c.reconcileSecret(ctx, &secrets.Items[i], pinnipedInfoCM, log); err != nil {
-				log.Error(err, "error reconciling secret")
+				log.Error(err, "error reconciling secret", "secret", secrets.Items[i].Name, "namespace", secrets.Items[i].Namespace)
```

Comment:
We log the more specific errors already within the `reconcileSecret` function or the `reconcileDataValues` function.  Not sure we need to log the error again since we're not adding anything new.